.devcontainer/emscripten-version.txt

```diff
@@ -0,0 +1 @@
+3.1.12
```

Comment:
We'll almost certainly forget to bump this when we bump emscripten the next time 😄 Can you please add a comment here which points to this file, hopefully that should catch it:
https://github.com/dotnet/icu/blob/5769c2dc1db6a1da43d7b0342899aa03b097da8f/eng/azure-pipelines.yml#L43